README.md

```diff
@@ -60,6 +60,15 @@ config :ethereumex,
   ipc_path: "/path/to/ipc"
 ```
 
+If you want to count the number of RPC calls per RPC method,
+set adapter in the configuration. The adapter needs to implement `increment/2`
+where the first argument is the key, the second in the count integer.
+
+```elixir
+config :ethereumex,
+  adapter: Datadog
```

Comment:
Can you please add an example of the adapter in README? 
> the first argument is the key

Also, can you please clarify it? It looks like the first argument is the method name, right?

Reply:
added some text, pls check if it's better.... was struggling a bit :D